### PR TITLE
Fix publisher error caused by

### PR DESCRIPTION
**Pull request by tuxinaut**
Fixes

```
CREATE_FAILED Unable to complete request: ServiceInternal: Publisher error caused by: ServiceInternal: Publisher error
caused by: AccessDenied: User: arn:aws:sts::xxxxxx:assumed-role/uluru-managed-execution-role/66097711-xxxx-xxxx-xxxx-xxxxxxxxx is not authorized to perform: cloudwatch:PutMetricData
```

### CODE REVIEW

File: iam-openidconnectprovider/cmd/resource/model.go [hunk 1]
```diff
@@ -1,7 +1,5 @@
 package resource
 
-import "github.com/aws-cloudformation/cloudformation-cli-go-plugin/cfn/encoding"
-
 /*
 This file is autogenerated, do not edit;
```
Comment (ofosos) [inline iam-openidconnectprovider/cmd/resource/model.go:4]: Why do we have this in VC?

File: iam-openidconnectprovider/cmd/main.go [hunk 2]
```diff
@@ -3,6 +3,7 @@ package main
 import (
```
Comment (ofosos) [inline iam-openidconnectprovider/cmd/main.go:3]: Autogenerated, why in VC?

**Review by ofosos — COMMENTED** [review]
There are a couple of autogenerated files in VC. Is there a good reason to have these in VC?

**Comment by ofosos** [comment]
Apparently this is the way to go.

File: iam-openidconnectprovider/cmd/resource/model.go (hunk 1 above)
Comment (tuxinaut) [reply iam-openidconnectprovider/cmd/resource/model.go:4]: This file is an essential part of the project and shall be available even without `cfn generate` run. Without this file, in version control, upstream changes won't be visible.

File: iam-openidconnectprovider/cmd/main.go (hunk 2 above)
Comment (tuxinaut) [reply iam-openidconnectprovider/cmd/main.go:3]: Same as above.

**Review by ofosos — APPROVED** [review]
Looks good to me.

**Review by kalrish — APPROVED** [review]
Looks good.

**Pull request merged**